evm-contracts/src/v0.6/examples/VRFD20.sol

```diff
@@ -0,0 +1,138 @@
+pragma solidity 0.6.6;
+
+import "../VRFConsumerBase.sol";
+import "../Owned.sol";
+
+/**
+ * @notice A Chainlink VRF consumer which uses randomness to mimic the rolling
+ * of a 20 sided die
+ */
+contract VRFD20 is VRFConsumerBase, Owned {
+    using SafeMathChainlink for uint;
```

Comment:
```suggestion
```
I believe the SafeMath behavior is already inherited from VRFConsumerBase. I believe we can remove this with zero consequences.

Reply:
Side note: I think we should always prefer the explicit `uint256` over the implied `uint`. This has bitten me before when calculating function sigantures.